src/Tribe/RSVP.php

```diff
@@ -255,6 +263,60 @@ public function register_types() {
 	}
 
 	/**
+	 * Update the RSVP values for this user
+	 */
+	public function update_attendee_data( $data, $order_id, $event_id ) {
+		$user_id = get_current_user_id();
+
+		$rsvp_orders = Tribe__Tickets__Tickets_View::instance()->get_event_rsvp_attendees( $event_id, $user_id );
+		$rsvp_order_ids = wp_list_pluck( $rsvp_orders, 'order_id' );
+
+		// This makes sure we don't save attendees for orders that are not from this current user and event
+		if ( ! in_array( $order_id, $rsvp_order_ids ) ) {
+			return;
+		}
+
+		// Get the Attendee Data, it's important for testing
+		foreach ( $rsvp_orders as $test_attendee ) {
+			if ( $order_id === $test_attendee['order_id'] ){
+				continue;
+			}
+
+			$attendee = $test_attendee;
```

Comment:
Is it possible that `$attendee` might never be set, in some scenarios? If so should we test if it is set following the foreach block before trying to access indicies for that var?